Automattic/wp-calypso
# Launchpad: Adds the new Site Setup to the card components list

Related to #84409

## Proposed Changes

* We are migrating the Site Setup checklist to use the Launchpad. 
* This PR adds the components and settings necessary to render Launchpad for the new checklist that we are calling `legacy-site-setup`.  The component definition was also added on the back in this diff: D130311-code

<img width="729" alt="Screen Shot 2023-12-14 at 14 15 07" src="https://github.com/Automattic/wp-calypso/assets/1234758/5138a90f-5dcd-42e5-8bfb-404b1c2a223c">

## Testing Instructions



* Create a new site through the `/setup/free` flow.
* Go through the flow until you reach the Fullscreen launchpad, and Launch your site without completing any tasks
* You should land on the Customer Home with a banner inviting you to show the `Site setup`, click on the `Show site setup` button
* Make sure you are sandboxed
* Edit your `0-sandbox.php` file, adding the following line:
```
add_filter( 'enable_legacy_site_setup_launchpad', '__return_true' );
```
* Refresh the Customer Home page, and you should see the Launchpad component


## Pre-merge Checklist
